Resources/Prototypes/Datasets/Names/fortunes.yml

```diff
@@ -0,0 +1,82 @@
+- type: dataset
+  id: CookieFortuneDescriptions
+  values:
+  - The end is near... and it's all your fault.
+  - We know what you did.
+  - Beware.
+  - Don't look back.
+  - Never trust a person in squeaky shoes.
+  - Never trust a person in red uniform.
+  - Never trust a person in green uniform.
+  - Never trust a person in blue uniform.
+  - Never trust a person in yellow uniform.
+  - Never trust a person in black uniform.
+  - Never trust a person without uniform.
+  - Always trust a person in squeaky shoes.
+  - Clown knows that you know that he knows. Beware.
+  - Beware of silent man in silent maints.
```

Comment:
```suggestion
  - Beware of the silent man in the silent maintenance tunnels.
```

Reply:
This flows worse, why?

Reply:
Because it seems like incorrect grammar to me.

Reply:
English is not my first language, so I'll rely on Visne suggestions;
Many thanks.